core-kotlin-modules/core-kotlin-lang-2/README.md

```diff
@@ -11,5 +11,6 @@ This module contains articles about core features in the Kotlin language.
 - [Lazy Initialization in Kotlin](https://www.baeldung.com/kotlin-lazy-initialization)
 - [Comprehensive Guide to Null Safety in Kotlin](https://www.baeldung.com/kotlin-null-safety)
 - [Kotlin Scope Functions](https://www.baeldung.com/kotlin-scope-functions)
+- [If-Else Expression in Kotlin](https://www.baeldung.com/kotlin/if-else-expression)
```

Comment:
Why is this change part of this PR?

Reply:
I worked on this article before. Since the link to the article doesn't exist yet, so I added it as a part of this PR.

Reply:
fair enough :)